.dockerignore

```diff
@@ -0,0 +1,3 @@
+.git
```

Comment:
I'm not sure if this useful... as it's `bldr` based, I believe this has no effect. (and `bldr` never loads whole directory)